pkg/cmd/server/api/validation/node.go

```diff
@@ -28,9 +28,15 @@ func ValidateNodeConfig(config *api.NodeConfig, fldPath *field.Path) ValidationR
 	}
 	validationResults.AddErrors(ValidateKubeConfig(config.MasterKubeConfig, fldPath.Child("masterKubeConfig"))...)
 
+	if len(config.DNSBindAddress) > 0 {
+		validationResults.AddErrors(ValidateHostPort(config.DNSBindAddress, fldPath.Child("dnsBindAddress"))...)
+	}
 	if len(config.DNSIP) > 0 {
 		validationResults.AddErrors(ValidateSpecifiedIP(config.DNSIP, fldPath.Child("dnsIP"))...)
 	}
+	for i, nameserver := range config.DNSNameservers {
```

Comment:
I guess not... just don't want people (more) confused. not a big deal either way